renormalizer/property/ops.py

```diff
@@ -0,0 +1,64 @@
+from renormalizer.mps import Mpo
+from renormalizer.utils import Quantity
+from renormalizer.model import MolList
+
+import numpy as np
+
+
+def e_ph_static_correlation(mol_list: MolList, imol:int =0, jph:int =0,
+        periodic:bool =False, name:str="S"):
+    '''
+    construct the electron-phonon static correlation operator in polaron problem
+    if periodic:
+        S_(m, jph) = \frac{1}{D} \sum_n \langle x_{m+n,jph} a_n^\dagger a_n \rangle
```

Comment:
What is the definition of `D`? Is this implying periodic *homogeneous* model?

Reply:
D is the displacement. I will add a reference paper.